blocks/library/gallery/block.scss

```diff
@@ -0,0 +1,39 @@
+
+.blocks-gallery {
```

Comment:
If we do this, it will break on the editor side of things. Going to keep as `.blocks-gallery` for now.